custom/pnlppgi/expressions.py

```diff
@@ -8,7 +8,10 @@ class WeekExpressionSpec(JsonObject):
     property = StringProperty()
 
     def __call__(self, item, context=None):
-        date = force_to_datetime(item[self.property])
+        try:
+            date = force_to_datetime(item[self.property])
+        except ValueError:
+            return -1
```

Comment:
@czue I don't know why but in data we have strange value for ```date_fin``` property. for example ```43256.0```.   ```-1``` is not used  in reports but we can create report and check how many records have this value and probably update the records.